SPECS/libcap/libcap.spec

```diff
@@ -32,15 +34,18 @@ make prefix=%{_prefix}	SBINDIR=%{_sbindir} PAM_LIBDIR=%{_libdir} RAISE_SETFCAP=n
 %ifarch aarch64
 mv %{buildroot}%{_libdir} %{buildroot}%{_lib64dir}
 %endif
-chmod -v 755 %{buildroot}/usr/lib64/libcap.so
+chmod -v 755 %{buildroot}%{_lib64dir}/libcap.so
+
 %check
 cd progs
 sed -i "s|pass_capsh --chroot=\$(/bin/pwd) ==||g" quicktest.sh
 ./quicktest.sh
+
 %files
 %defattr(-,root,root)
 %license License
 %{_lib64dir}/libcap.so.*
+%{_lib64dir}/libpsx.so.*
```

Comment:
Sounds good! I've updated the PR to install the libraries into `%{_libdir}` instead of `%{_lib64dir}` and it builds clean locally. I'll run a build through the pipelines just to be sure.

Reply:
Pipeline builds built clean. ARM64 build had an issue with an unrelated package (`zfs-fuse` which has been fixed already in `main`)